portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+
+      <header>
+          <nav>
+               <ul>
+    <li><a href="index.html">Home</a></li>
+    <li><a href="about.html">About Me</a></li>
+    <li><a href="resume.html">Resume</a></li>
+    <li><a href="activities.html">Activites</a></li>
+    <li><a href="accademic.html">Academic Achievements</a></li>
+    <li><a href="contact.html">Contact Us</a></li>
```

Comment:
Indentation: 2 spaces ahead of <ul></ul> level.

Reply:
Yes here's the link to the website.

Website: http://jdrummond-sps-summer21.appspot.com/index.html

Reply:
Can you please indent the <li> to be 2 spaces ahead of <ul>...

e.g.:
```
<header>
  <nav>
    <ul>
      <li><a href=...
      <li><a href=...
    </ul>
```